# file: C:\Users\micha\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\_pytest\debugging.py
# hypothesis_version: 6.138.15

['--pdb', '--pdbcls', '--trace', '.', ':', '>', 'PDB continue', 'Quitting debugger', '__tracebackhide__', 'all', 'captured ', 'capturemanager', 'do_continue', 'do_debug', 'do_quit', 'entering PDB', 'general', 'global', 'header', 'log', 'modulename:classname', 'pdbinvoke', 'pdbtrace', 'post_mortem', 'runcall', 'set_trace', 'stderr', 'stdout', 'store_true', 'terminalreporter', 'trace', 'traceback', 'usepdb', 'usepdb_cls']